Python/Product/Analysis/Interpreter/IPackageManager.cs

```diff
@@ -52,6 +52,35 @@ public interface IPackageManager {
         event EventHandler IsReadyChanged;
 
         /// <summary>
+        /// Name to display for the extension in the environments window.
+        /// Example: 'Packages (PyPI)'
+        /// </summary>
+        string ExtensionDisplayName { get; }
```

Comment:
I assume this is localized text? Do we need an unlocalized version (e.g. for settings)?

Reply:
Not at this time, but I see what you're saying. I can add one but it would be unused for now.

Reply:
Let's just clarify that it should be localized in the comment.